apps/palette-tauri/src/styles.css

```
@@ -2232,29 +2552,47 @@ textarea {
+.ask-activity-row:last-child::before {
+  display: none;
+}
+
 .ask-activity-row svg {
+  position: relative;
+  z-index: 1;
   margin-top: 1px;
+  padding: 2px;
   color: var(--aurora-accent-pink-strong);
+  background: color-mix(in srgb, var(--aurora-panel-medium) 88%, transparent);
+  border: 1px solid color-mix(in srgb, currentColor 22%, var(--aurora-border-default));
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)